packages/docusaurus/lib/commands/build.js

```diff
@@ -18,9 +19,14 @@ const createServerConfig = require('../webpack/server');
 const createClientConfig = require('../webpack/client');
 const {applyConfigureWebpack} = require('../webpack/utils');
 
-function compile(config) {
+function compile(config, isServer) {
```

Comment:
Is this function used outside of this file? Wondering if there's really a need for `isServer` since it's always set to `true`.

Reply:
Just a wrapper function for webpack. 

see
https://github.com/facebook/Docusaurus/blob/d929bf78002d6130bec7c1cb0cdb7fed7951c435/packages/docusaurus/lib/commands/build.js#L87-L93

Reply:
Ah I see. Good call.